forest/sycamore/survey_config.py

```diff
@@ -22,75 +23,117 @@ def convert_time_to_date(submit_time, day, time):
         time(list):
             List of timings times from the configuration surveys information
     """
-
     # Convert inputted desired day into an integer between 0 and 6
     day = day % 7
     # Get the days of the given week using the dow of the given submit day
     dow = submit_time.weekday()
-    days = [submit_time + datetime.timedelta(days=i) for i in range(0 - dow, 7 - dow)]
+    days = [submit_time + datetime.timedelta(days=i)
+            for i in range(0 - dow, 7 - dow)]
 
     time = [str(datetime.timedelta(seconds=t)) for t in time]
     time = [t.split(':') for t in time]
     time = [[int(p) for p in t] for t in time]
 
     # Get rid of timing
     #     https://stackoverflow.com/questions/26882499/reset-time-part-of-a-pandas-timestamp
-    #     print(time)
     days = [d - pd.offsets.Micro(0) for d in days]
-    days = [[d.replace(hour=t[0], minute=t[1], second=t[2], microsecond=0) for t in time] for d in days]
+    list_of_days = [[d.replace(hour=t[0], minute=t[1], second=t[2],
+                               microsecond=0) for t in time] for d in days]
 
-    return days[day]
+    return list_of_days[day]
 
 
-def generate_survey_times(time_start, time_end, timings=[], survey_type='weekly'):
-    """
-    Takes a start time and end time and generates a schedule of all sent surveys in time frame for the given survey type
+def generate_survey_times(
+        time_start: str,
+        time_end: str,
+        timings: Optional[list] = None,
+        survey_type: str = 'weekly',
+        intervention_dict: Optional[dict] = None) -> list:
```

Comment:
Closing and opening parentheses should match. Also, there's an extra indent for all args.

Reply:
This is updated